src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/AsylumCaseFieldDefinition.java

```diff
@@ -1011,6 +1011,9 @@
     HOME_OFFICE_ADJOURN_WITHOUT_DATE_INSTRUCT_STATUS(
         "homeOfficeAdjournWithoutDateInstructStatus", new TypeReference<String>() {}),
 
+    HOME_OFFICE_CHANGE_DIRECTION_DUE_DATE_INSTRUCT_STATUS(
```

Comment:
Do we not need to differentiate between evidence bundle and request review changes?

Reply:
Changed to have different status